app/allocation/controllers/view.js

```diff
@@ -1,38 +1,51 @@
+const { get, sortBy } = require('lodash')
+
+const permissions = require('../../../common/middleware/permissions')
 const presenters = require('../../../common/presenters')
 
 module.exports = function view(req, res) {
   const { allocation } = res.locals
-  const { moves } = allocation
+  const { moves, status } = allocation
+  const userPermissions = get(req.session, 'user.permissions')
   const bannerStatuses = ['cancelled']
-
   const movesWithoutPerson = moves.filter(move => !move.person)
+  const movesWithPerson = moves.filter(move => move.person)
   const moveToCardComponent = presenters.moveToCardComponent({
     showStatus: false,
   })
 
+  const removeUnassignedMoves = move => {
+    return !(
+      permissions.check('allocation:person:assign', userPermissions) &&
+      !move.person
+    )
+  }
+
   const locals = {
```

Comment:
I know some of us value terseness (and ternaries, ternaries, ternaries...) but I find it really hard to see what's actually being exported to locals with all the inline assignments. Not just here (and in fact this is hardly the most egregious example) but I just thought I'd mention it

Reply:
Yeah, I do see what you mean. Especially with files like this where there are a number of things being passed down to the view.

I wonder if we try to adopt a pattern where  the object passed to the render method always uses shorthand variable names and they are defined using variables within the controller to aid readability. What do you think?

Although, of course, there could be exception to the rule if they are simple assignments such as `query: req.query`.